packages/alert/index.js

```diff
@@ -47,44 +25,56 @@ class WarpAlert extends LitElement {
 
   connectedCallback() {
     super.connectedCallback();
-    if (!this.variant || !Object.keys(variants).includes(this.variant)) {
+    if (!this.variant || !Object.keys(icons).includes(this.variant)) {
       throw new Error(
-        'Invalid "variant" attribute. Set its value to one of the following:\nnegative, positive, warning, info.',
+        'Invalid "variant" attribute. Set its value to one of the following:\nnegative, positive, warning, info.'
       );
     }
   }
 
+  get _wrapperClasses() {
+    return classNames({
+      [ccAlert.alert]: true,
+      [ccAlert[this.variant]]: true,
+    });
+  }
+
+  get _iconClasses() {
+    const activeIconClassNames = ccAlert[`${this.variant}Icon`];
+
+    return classNames({
+      [ccAlert.icon]: true,
+      [activeIconClassNames]: true,
+    });
+  }
+
   // Slotted elements remain in lightDOM which allows for control of their style outside of shadowDOM.
   // ::slotted([Simple Selector]) confirms to Specificity rules, but (being simple) does not add weight to lightDOM skin selectors,
   // so never gets higher Specificity. Thus in order to overwrite style linked within shadowDOM, we need to use !important.
   // https://stackoverflow.com/a/61631668
-  static styles = 
-    css`
-      /* @unocss-placeholder */
-      :host {
-        display: block;
-      }
-      ::slotted(:last-child) {
-        margin-bottom: 0px !important;
-      }
-    `;
+  static styles = css`
+    /* @unocss-placeholder */
+    :host {
+      display: block;
+    }
+    ::slotted(:first-child) {
+      margin-top: 0px;
+    }
+    ::slotted(:last-child) {
+      margin-bottom: 0px !important;
+    }
+  `;
 
-  get _style() {
-    return this.variant ? variants[this.variant] : {};
+  get _icon() {
+    return this.variant ? icons[this.variant] : {};
   }
 
   render() {
-    const { color, icon, bgColor, borderColor, textColor } = this._style;
-
     return html`
       <w-expand-transition ?show=${this.show}>
-        <div
-          role=${this.role}
-          class="${`flex p-16 border rounded-4 border-l-4 ${bgColor} ${borderColor}`}"
-          style="border-left-color:var(--w-${color}-600)"
-        >
-          <div class="mr-8 ${textColor}">${icon}</div>
-          <div class="text-14">
+        <div role=${this.role} class=${this._wrapperClasses}>
+          <div class=${this._iconClasses}>${this._icon}</div>
+          <div class="last:mb-0 text-14">
```

Comment:
Ah, these classes should also be exported from component-classes and updated. Could you add a TODO here so we have some way of remembering it? 🙏